make-webpack-config.js

```diff
@@ -25,6 +26,7 @@ module.exports = function(options){
   }
 
   return {
+    devtool: isProd ? 'hidden-source-map' : 'eval',
```

Comment:
@FredyC This is in reaction to https://github.com/KeywordBrain/redbox-react/issues/53, so there might be a use case for it. That's why I used `hidden-source-map` first, because it does not impact production and you'd actively have to add the source map to your Chrome DevTools to make it use them.